### PR TITLE
Add p0-input-image

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
Add: add p0-input-image

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge

**Comment by sri-rishi** [comment]
Look good to merge

**Comment by anasm11** [comment]
The code looks good. The input box cant be recognised at first glance, maybe adding border would help.

**Review by sri-rishi — APPROVED** [review]
Looks good to merge

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.img_res {
+  height: 100%;
+  width: 100%;
+}
```
Comment (Ankur9669) [inline components/image/image.css:5]: You can add object-fit property here

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  width: 100%;
+}
+.inp_md {
+  height: 2rem;
+  width: 100%;
+}
+.inp_sm {
+  height: 1rem;
+  width: 100%;
+}
```
Comment (Ankur9669) [inline components/input/input.css:26]: All your inputs are of width:100%.
So you can add width: 100% to you basic_inp class instead of adding everywhere.

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The UI and code looks great, just do these changes and it would be great to merge

**Comment by gauravsinhaweb** [comment]
> The code looks good. The input box cant be recognised at first glance, maybe adding border would help.

Updated the changes, thanks!

**Comment by gauravsinhaweb** [comment]
> 

Updated the changes, thanks!

**Pull request merged**